Sources/DistributedActors/ClusterSystem.swift

```diff
@@ -491,8 +497,10 @@ public class ClusterSystem: DistributedActorSystem, @unchecked Sendable {
         self.shutdownSemaphore.wait()
 
         /// Down this member as part of shutting down; it may have enough time to notify other nodes on an best effort basis.
-        if let myselfMember = self.cluster.membershipSnapshot.uniqueMember(self.cluster.uniqueNode) {
-            self.cluster.down(member: myselfMember)
+        Task {
+            if let myselfMember = try await self.cluster.membershipSnapshot.uniqueMember(self.cluster.uniqueNode) {
+                self.cluster.down(member: myselfMember)
+            }
         }
```

Comment:
This probably is enough to use the `cluster.down(node:)` API, so then we don't need the task and async await dance 👍

Reply:
amended https://github.com/apple/swift-distributed-actors/pull/969/commits/3efd08ffa64e97123a35b9c90d5e9290e700ed87